packages/core/editor/src/editor/blocks/toggleBlock.ts

```diff
@@ -84,6 +84,16 @@ export function toggleBlock(editor: YooEditor, toBlockTypeArg: string, options:
 
   // [TEST]
   if (options.deleteText) {
+    editor.updateBlock(newBlock.id, {
+      meta: newBlock.meta,
+      value: [
+        {
+          ...newBlock.value[0],
+          children: [{ text: '' }],
+        },
```

Comment:
It can break logic for complex blocks with multiple nested Slate children.
My suggesstion: 
```tsx
    const lastNode = Editor.last(newSlate, [0]);
    Transforms.delete(newSlate, { at: lastNode[1] });
```